etc/sample.openstack.yaml

```diff
@@ -73,6 +73,8 @@ compute:
         ops:
             # the project id in OpenStack
```

Comment:
Comment should be changed to "the project name in OpenStack"
But see comment above